openage/convert/driver.py

```diff
@@ -279,7 +282,7 @@ def convert_media(args):
             # do the dir "renaming"
             if dirname == "gamedata" and filepath.suffix == ".slp":
                 output_dir = "graphics"
-            elif dirname == "gamedata" and filepath.suffix == ".wav":
+            elif dirname == "interface" and filepath.suffix == ".wav":
```

Comment:
I have `.wav` files in my gamedata and interface mounts (see for yourself with `python3 -m openage convert -i`).
That condition should therefore be `elif dirname in {"interface", "gamedata"} and filepath.suffix = ".wav"` I'd say/